server/lib/next.ts

```diff
@@ -0,0 +1,20 @@
+import * as express from 'express';
+import next from 'next';
+
+const isDev = process.env.NODE_ENV !== 'production';
```

Comment:
Maybe could be interesting to change this line to this:

```js
const isDev = process.env.NODE_ENV == 'development';
```
On the nexts  PR's we'll have also a testing environment.

Reply:
I think `===` is always preferred over `==` in a production environment for type safety.  Regarding other types of environment, this `dev: isDev` gets passed to the next.js config, which for the purposes of development vs production vs staging vs testing (or any other env we might use), anything that is not production should be development. Let me know if I'm missing something!

Reply:
I didn't know about this Nextjs detail. You're right.